releng/target-platform-4Q2023/target-platform-4Q2023.target

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<!--
+  Copyright (c) 2023 IBM Corporation and others.
+  
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Contributors:
+      IBM Corporation - initial implementation
+-->
+<target includeMode="feature" name="target-platform.target" sequenceNumber="15">
+    <locations>
+        <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <repository location="https://download.eclipse.org/releases/2023-12/"/>
+            <!-- The following units will use versions included in the Eclipse 4Q2023 release -->
+            <unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
```

Comment:
What is the reason for switching to 0.0.0?  Is it just to avoid having to go through the trouble of listing out versions unit-by-unit?  I thought the Eclipse PDE tools helped us with this.   I guess it's maybe useful to have this as documentation ( a record) of the versions we're testing against, e.g. even for something like the lsp4e-related versions.   I don't think it hugely matters in that I'm not sure the contents are ever updated (are they???  Do we know that 100% for a fact)?